src/vcf/assembly_checker.cpp

```diff
@@ -63,9 +73,14 @@ namespace ebi
 
               for (auto &output : outputs ) {
                   if (match_result) {
-                      output->write_match(record_core);
+                      std::string match_result = "Line " + std::to_string(line_num) + ": Reference allele \'"
+                          + reference_sequence + "\' matches with the fasta_sequence";
+                      output->match(record_core,match_result);
                   } else {
-                      output->write_mismatch(record_core);
+                      std::string mismatch_result = "Line " + std::to_string(line_num) + ": Reference allele \'"
```

Comment:
putting the chromosome and position would also be helpful here and in the previous message:

Line 100: (Chromosome 1, position 100) reference allele 'A' does not ...

Reply:
Suggested rewording:

> Line 100: Chromosome 1, position 100, reference allele 'A' does not match the reference sequence, expected 'G'

Reply:
done